ecl2df/gruptree.py

```diff
@@ -225,10 +223,44 @@ def edge_dataframe2dict(dframe):
 
     children, parents = zip(*edges)
     roots = set(parents).difference(children)
-    trees = []
-    for root in list(roots):
-        trees.append({root: subtrees[root] for root in roots})
-    return trees
+    return [{root: subtrees[root]} for root in sorted(roots)]
+
+
+def _sub_tree_from_dict(nested_dict, name):
+    """Internal recursive function for generating trees."""
+    assert isinstance(nested_dict, dict)
+    tree = treelib.Tree()
+    tree.create_node(name, name)
+    child_names = list(nested_dict.keys())
+    child_names.sort()
+    for child_name in child_names:
+        # The paste() function requires unique node names.
+        tree.paste(name, _sub_tree_from_dict(nested_dict[child_name], child_name))
+    return tree
+
+
+def tree_from_dict(nested_dict):
+    """Convert a dictonary to a treelib Tree.
+
+    The treelib representation of the trees is used
+    for pretty-printing (in ASCII) of the tree, you
+    only need to do str() of the returned result
+    from this function.
+
+    See `https://treelib.readthedocs.io/`
+
+    Args:
+        nested_dict: nested dictonary representing a tree.
+
+    Returns:
+        treelib.Tree
+    """
+    if not nested_dict:
+        # Return an empty string, because str(treelib.Tree()) will error
+        return ""
+    assert len(nested_dict.keys()) == 1, "One tree at a time please"
```

Comment:
Should we provide a bit more feedback to the user? It might not be obvious what "one tree at a time" means

Reply:
Fixed.